experimental/dds/tree2/.dependency-cruiser-known-violations.json

```diff
@@ -0,0 +1,727 @@
+[
+  {
+    "type": "cycle",
+    "from": "../../../packages/dds/test-dds-utils/dist/ddsFuzzHarness.js",
```

Comment:
Since we are excluding `node_modules`, I think it makes sense to exclude other packages that are within the repo to avoid reporting errors in them here.